pkg/util/iptables/iptables.go

```diff
@@ -162,6 +163,9 @@ var MinCheckVersion = utilversion.MustParseGeneric("1.4.11")
 // used for port mapping to be fully randomized
 var RandomFullyMinVersion = utilversion.MustParseGeneric("1.6.2")
 
+// RandomFullyMinKernelVersion is the minimum kernel version from which the --random-fully flag is supported,
+var RandomFullyMinKernelVersion = utilversion.MustParseGeneric("3.13")
```

Comment:
Can we add a reference in the comments for when this patch went in?

Reply:
done, PTAL